Python/ceval.c

```diff
@@ -4851,6 +4855,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             if (res == NULL) {
                 goto error;
             }
+            CHECK_EVAL_BREAKER();
```

Comment:
This is a fast operation, no need to check the eval-breaker.

Reply:
Fair enough. I tried to think of a contrived example with a complex `__len__`, but then realised anything that complex definitely has other opportunities to call `CHECK_EVAL_BREAKER` in jump or call opcodes.